lib/cmd/license-cmd.js

```diff
@@ -3,28 +3,31 @@
 
 const program = require('commander');
 const ThirdPartyLicenseGenerator = require('./third-party-license-generator');
+const LicenseFileHandler = require('./license-file-handler');
 
 program
-  .option('--header <value>', 'Add a header to license file.')
-  .option('--allow <value>', 'Comma separated list of allowed license names to be verified.')
-  .option('--file <value>', 'File to write the license list.')
-  .option('--skipPrefix <value>', 'Skip licenses of dependencies that start with this')
+  .option('--config <value>', 'Use the config file.')
   .parse(process.argv);
 
-let header = program.header;
-let allowedLicenseList = program.allow.split(',');
-let file = program.file;
-let skipPrefix = program.skipPrefix;
+let config = null;
+
+LicenseFileHandler
+  .getFile(program.config ? program.config : '.license-config')
+  .then((content) => {
+    return LicenseFileHandler.parseFile(content);
+  })
+  .then((json) => {
+    config = json;
+  })
+  .catch((error) => {
+    console.error(error.errorMessage);
+    process.exit(1);
+  });
 
 ThirdPartyLicenseGenerator
   .getDependenciesDetails()
   .then((dependenciesDetails) => {
-    return ThirdPartyLicenseGenerator.createThirdPartyLicense(dependenciesDetails, {
-      header: header,
-      allowedLicenseList: allowedLicenseList,
-      file: file,
-      skipPrefix: skipPrefix
-    });
+    return ThirdPartyLicenseGenerator.createThirdPartyLicense(dependenciesDetails, config);
```

Comment:
This only works because LicenseFileHandler.getFile.... is synchronous. 
If it was async, we would have something like:
```
let promiseGetJsonFile = LicenseFileHandler..getFile(program.config ? program.config : '.license-config')  
  .then((content) => {
    return LicenseFileHandler.parseFile(content);
  })
let promiseGetDependenciesDetails = ThirdPartyLicenseGenerator.getDependenciesDetails()		    

return Promise.all([promiseGetJsonFile,promiseGetDependenciesDetails].then((items)=>{
  let resultPromiseGetJsonFile = items[0];
  let resultPromiseGetDependenciesDetails = items[1];
  
  return ThirdPartyLicenseGenerator.createThirdPartyLicense(
      resultPromiseGetDependenciesDetails , 
      resultPromiseGetJsonFile);
}).then.....//continue